site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/model/AddressConfiguration.java

```diff
@@ -0,0 +1,64 @@
+package uk.ac.ox.ndph.mts.site_service.model;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+/**
+ * Address configuration Model
+ */
+@Component
+@Configuration
+@ConfigurationProperties("address")
+public class AddressConfiguration {
```

Comment:
Would it be better to just use the FHIR Address model from HAPI `org.hl7.fhir.r4.model.Address`and not define a custom model? Then the "conversion" would just be deserializing/reserializing. The address gets mapped to a FHIR Address anyway. FHIR address appears to have all the required elements:
0-many lines of address
city (town)
district (county)
state (can be empty)
postalCode
country

Reply:
@alexbaxter, I thought we said we'll "hide"/abstract the FHIR implementation from the service upward to the UI...

Reply:
Agreed for "business" entities in general since Practitioners and Organizations would just confuse things in the API to the front end. I'm not sure it matters so much for Address since one implementation is much like another (if it supports the formatting we want).

Reply:
In this iteration lets leave a simple MTS Address model. Lets follow the same pattern of our MTS model.